utils/ioutils/promptstrings.go

```diff
@@ -85,7 +85,7 @@ func SelectString(items []PromptItem, label string, onSelect func(PromptItem)) e
 	return nil
 }
 
-// In MacOS, Terminal bell is ringing when trying to select items using up and down arrows.
+// In macOS, Terminal bell is ringing when trying to select items using up and down arrows.
```

Comment:
In macOS, Terminal bell is ringing when trying to select items using up and down arrows.
-->
On macOS the terminal's bell is ringing when trying to select items using the up and down arrows.